### PR TITLE
help.lisp: redesign front page.

### LINKED ISSUE #2668: Redesign start page
The startup page is not intuitive enough, even for those who might be comfortable with programs like Emacs.

Previously our startup page listed basic keybindings, but now it links to the tutorial where these can be found. The idea of using a panel buffer for that purpose has been suggested. 

Requirements:

- All actionable elements must be styled in a way that makes it obvious how to interact with them;
- Must introduce a handful of keybindings for the user to make its first steps; 
- Must be a variation of the new Nyxt's website start page (that @lansingthomas is designing).

CC @aartaka @lansingthomas @jmercouris

### PR DESCRIPTION
**Pull request by jmercouris**
Closes #2668 

Screenshot_20230720_140522

### CODE REVIEW

File: source/help.lisp [hunk 1]
```diff
@@ -172,85 +172,82 @@ The value is saved to clipboard."
...
-      (:div (:nbutton :text "Start searching!"
-              '(set-url :prefill-current-url-p nil)))
-      (:div (:nbutton :text "Quick Start"
-              '(quick-start))))
+      (:nav
+       (:nbutton :text "Quick-Start"
+                 '(quick-start))
+       (:a :class "button" :href (nyxt-url 'describe-bindings)
+           :title "List all bindings for the current buffer."
+           "Describe bindings")
```
[the remark this answers is not in the document]
Comment (jmercouris) [reply source/help.lisp:232]: Acknowledged!

File: source/help.lisp [hunk 2]
```diff
@@ -172,85 +172,82 @@ The value is saved to clipboard."
...
+       (:a :class "button" :href (nyxt-url 'manual)
+           :title "Full documentation about Nyxt, how it works and how to configure it."
+           "Manual")
+       (:a :class "button" :href (nyxt-url 'common-settings)
+           :title "Switch between Emacs/vi/CUA key bindings, set home page URL, and zoom level."
+           "⚙ Settings"))
+      (:div :class "main"
```
[the remark this answers is not in the document]
Comment (jmercouris) [reply source/help.lisp:239]: Correct, you are right, fixed!

**Comment by jmercouris** [comment]
Screenshot_20230720_183732

**Comment by jmercouris** [comment]
The logo is generated with HTML, it is not the actual image, as that is not literally bundled in our app. I suppose we could use the literal image, but then we wouldn't be able to change the fill color easily. If we were to redo our image as an SVG, then we could.

**Comment by jmercouris** [comment]
Screenshot_20230720_184050

**Comment by jmercouris** [comment]
OK, i think it is ready!

**Comment by lansingthomas** [comment]
- [ ] `Describe-Bindings` button needs a big "B"
- [ ] `Set-URL` button and ` Execute-Command` button text needs to come down a few pixels.
> Screenshot 2023-07-20 at 10 55 14 AM
- [ ] the tile is forked up
   - I emailed you some fresh icons

**Comment by lansingthomas** [comment]
> The logo is generated with HTML, it is not the actual image, as that is not literally bundled in our app. I suppose we could use the literal image, but then we wouldn't be able to change the fill color easily. If we were to redo our image as an SVG, then we could.

SVGs in your inbox

File: source/help.lisp [hunk 3]
```diff
@@ -172,85 +172,82 @@ The value is saved to clipboard."
...
+            (:div :class "logo" "ny" (:br) "xt")
+            (:div
+             (:div (:nbutton :class "set-url" :text "Set-URL 🡪"
+                             '(set-url :prefill-current-url-p nil)))
+             (:div (:nbutton :class "execute-command" :text "Execute-Command 🡪"
+                             '(execute-command))))))
```
[the remark this answers is not in the document]
Comment (lansingthomas) [reply source/help.lisp:251]: Lets see... removing border and resizing.

constant width for CTA experiment

Yeah I like it. Seems very orderly which is appropriate here.
good call @aadcg 
- [ ] remove border for `Set-URL`
- [ ] resize the button to match width

**Comment by jmercouris** [comment]
I got the SVGs I'm just not sure how to include them in the bundle :-O

**Comment by jmercouris** [comment]
Screenshot_20230720_200139

**Comment by jmercouris** [comment]
I did achieve it with the SVG, I just don't like how I did it. Suggestions are welcome; @atlas-engineer/atlas

**Comment by lansingthomas** [comment]
Looking great John!
can we get the CTA buttons to align with the tile? :D 

like this:
Screenshot 2023-07-20 at 1 11 36 PM

File: source/global.lisp [hunk 4]
```diff
@@ -166,3 +166,20 @@ Return nil on error."
...
+<path d=\"M116.5 155L92.5 187.5L85 178.5L102.5 155H116.5Z\" fill=\"white\"/>
+<path d=\"M119 229L63.5 155H49.5L76 190.5L47 229H61.5L83 200L104.5 229H119Z\" fill=\"white\"/>
+<path d=\"M147 164V154.5H167V135L179.5 130V154.5H207V164H179.5V207C179.5 215 183.5 219.5 190.5 219.5H207V229H190.5C172 229 167 219 167 207V164H147Z\" fill=\"white\"/>
+</g>
+<defs>
+<clipPath id=\"clip0_1076_1322\">
+<rect width=\"256\" height=\"256\" fill=\"white\"/>
+</clipPath>
+</defs>
+</svg>")
```
[the remark this answers is not in the document]
Comment (jmercouris) [reply source/global.lisp:185]: I did consider it, just not sure how to properly resolve that path at runtime, this does give me an idea though, and I can fix this. Thanks Andre!

**Comment by jmercouris** [comment]
Screenshot_20230721_191744

**Comment by jmercouris** [comment]
Ready for final code review!

**Comment by jmercouris** [comment]
You are welcome, I am happy to prrrrrrovide.

**Pull request merged**